### ISSUE TITLE
PyInstaller can't properly bundle pandas_ta on windows

### ISSUE THREAD

**Issue by kolahghermezi** [twopirllc/pandas-ta#151]
**Which version are you running? The lastest version is on Github. Pip is for major releases.**
* Version: 0.2.23b0

**Upgrade.**
* Already done.

**Describe the bug**
When I bundle my script using PyInstaller, Pyinstaller creates the bundled EXE just fine. But when I want to open my EXE I'd get the following traceback:

Traceback (most recent call last):
  File "main.py", line 3, in <module>
  File "C:\Users\my_username\miniconda3\envs\talib\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
  File "pandas_ta\__init__.py", line 96, in <module>
  File "C:\Users\my_username\miniconda3\envs\my_env\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
  File "pandas_ta\core.py", line 12, in <module>
ImportError: cannot import name 'version' from 'pandas_ta' (C:\Users\my_username\AppData\Local\Temp\_MEI42442\pandas_ta\__init__.pyc)
[4208] Failed to execute script main

I've tried using onedir instead of onefile with no luck. Also tried using the official cpython+pip instead of miniconda and still no luck.

**To Reproduce**
1. Import pandas_ta in a project.
2. Try to create bundled EXE using PyInstaller.
3. It doesn't matter if you use onefile or not.
4. Try to run the bundled EXE file.

**Expected behavior**
* The bundled EXE would run without a traceback regarding pandas_ta.

**Comment by twopirllc** [comment]
Hello @kolahghermezi,

Thanks for filling out this bug report.

I do not use _onefile_, _anaconda_, _miniconda_, nor _PyInstaller_.

Could you go into the Pandas TA code in ```__init__.py``` and try changing this:
```python
else:
    version = __version__ = _dist.version
```

to
```python
else:
    __version__ = _dist.version

version = __version__
```

and tell me if that works.

Thanks,
KJ

**Comment by jkjns** [comment]
I have the same issue.
I don't know about @kolahghermezi but the suggested method by @twopirllc did not work.
The error is the same as @kolahghermezi's error.

**Comment by twopirllc** [comment]
@jkjns, @kolahghermezi  

Thanks for the update. If you find a fix, please fork, edit and make a pull request.

Thanks,
KJ

**Comment by twopirllc** [comment]
@jkjns 

Try removing the **else statement** from ```__init__.py```:
```python
else:
    version = __version__ = _dist.version
```
and remove the indent from the ```version``` line.
```python
version = __version__ = _dist.version
```

Please let me know how that works out.

Thanks,
KJ

**Comment by twopirllc** [comment]
@kolahghermezi @jkjns 

The latest version and bug fix #155 should take care of this issue.
```python
$ pip install -U git+https://github.com/twopirllc/pandas-ta
```

Let me know otherwise.

Thanks,
KJ
